### PR TITLE
Updated description of kubectl-gs login internal-api flag

### PR DESCRIPTION
**Pull request by vvondruska**
### What does this PR do?

Extended descriptoin of kubectl-gs login --internal-api flag to clarify tht it can be used with both management cluster and workload cluster API hostnames.

### Any background context you can provide?

The update reflects the latest changes in kubectl-gs, which newly supports deriving internal API hostname from workload cluster's main API URL.

### CODE REVIEW

File: src/content/ui-api/kubectl-gs/login.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ For **OIDC** authentication, the following flags are available:
 
 - `--token`: Use a bearer token instead of OIDC authentication, e. g. a service account token.
 
-- `--internal-api` - (AWS only) With this flag you use an internal Management API endpoint. It resolves to an internal IP address that is only accessible from within the cluster's virtual private cloud (VPC). The hostname of this endpoint is the same as the normal one, with the prefix `internal-`. Example: if your Management API host name is `g8s.example.yourdomain.tld`, the alternative hostname is `internal-g8s.example.yourdomain.tld`.
+- `--internal-api` - (AWS only) With this flag you use an internal Management API endpoint. It resolves to an internal IP address that is only accessible from within the cluster's virtual private cloud (VPC). The hostname of this endpoint is usually the same as the normal one, with the prefix `internal-`. Example: if your Management API host name is `g8s.example.yourdomain.tld`, the alternative hostname is `internal-g8s.example.yourdomain.tld`. This flag works with API hostnames of management clusters and workload clusters as well.
```
Comment (marians) [inline src/content/ui-api/kubectl-gs/login.md:75]:
```suggestion
- `--internal-api` - (AWS only) With this flag you use an internal Management API endpoint. It resolves to an internal IP address that is only accessible from within the cluster's virtual private cloud (VPC). The hostname of this endpoint is usually the same as the normal one, with the prefix `internal-`. Example: if your Management API host name is `g8s.example.yourdomain.tld`, the alternative hostname is `internal-g8s.example.yourdomain.tld`. This flag works with API hostnames of management clusters and workload clusters alike.
```

**Comment by marians** [comment]
Hi @vvondruska, do you still want to get this merged? If not, let me know and I can take over.

**Comment by pipo02mix** [comment]
can we merge this?

**Comment by marians** [comment]
@vvondruska What's the status of this? Can it be deleted, or should it actually get merged?

File: src/content/ui-api/kubectl-gs/login.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ user_questions:
   - How can I log in with kubectl for the Management API?
   - How can I create a workload cluster client certificate?
   - How do I specify the time to live for a workload cluster client certificate?
-last_review_date: 2022-05-20
+last_review_date: 2022-09-30
```
Comment (pipo02mix) [inline src/content/ui-api/kubectl-gs/login.md:18]:
```suggestion
last_review_date: 2023-07-11
```

**Comment by pipo02mix** [comment]
I am merging this since it is a small change and improve docs

**Pull request merged**